### PR TITLE
Drop fallback code for old GitHub Enterprise versions

### LINKED ISSUE #4863: CI is failing
WIP

### PR DESCRIPTION
**Pull request by fregante**
- Closes #4863

### CODE REVIEW

File: source/features/center-reactions-popup.css [hunk 1]
```diff
@@ -55,5 +54,5 @@ td.js-line-comments:last-child .timeline-comment-actions .add-reaction-popover::
 /* Keep colored button when reaction popup is open */
 .dropdown-details[open] .timeline-comment-action {
 	opacity: 100%;
-	color: var(--color-mktg-btn-bg, #4078c0); /* GHE #4121 */
```
Comment (fregante) [inline source/features/center-reactions-popup.css:58]: I also dropped every such comment. The idea was that fallback colors were just for GHE, but in reality GH keeps changing the class names…

cc @kidonng the class checker might not be too difficult to create:

1. Find `var([^,]+` in our repo
2. Fetch GitHub’s CSS file (if static)
3. Look for each variable

**Review by yakov116 — APPROVED** [review]
Untested but LGTM. I have a local branch with some more stuff but don't have any time to do it.

**Comment by yakov116** [comment]
I would not mark this as meta since old GHE users are affected.

**Comment by fregante** [comment]
Good idea

**Pull request merged**